src/Track.js

```diff
@@ -297,6 +298,28 @@ class Track {
 			trackEncoder.instrument(channelNumber, this.instrumentNumber)
 		}
 
+		const controlChangeKeys = Object.keys(this.controlChanges);
+
+		if (controlChangeKeys.length) {
+			const NB_MIDI_MESSAGES = 127;
+			let time = 0;
+
+			for (let key of controlChangeKeys) {
+				for (let control of this.controlChanges[key]) {
+					let midiEvent = new Encoder.Event({
+						type: Encoder.Event.CONTROLLER,
+						channel: channelNumber,
+						param1: control.number,
+						param2: control.value * NB_MIDI_MESSAGES,
+						time: control.time - time
```

Comment:
Anyway it's not working yet. The value for the time argument isn't correct. It should be in "ticks" instead of seconds. (control.time - time) * ticksPerSecond does not work, because it does not take in consideration the note events. I'm still figuring that out.